park-dashboard/src/Components/CampgroundInfo/CampgroundInformation.js

```diff
@@ -36,7 +40,11 @@ function CampgroundInformation(item) {
             </Typography>
           </CardContent>
           <CardActions>
-            <Button size="small" onClick={() => openUrl(item.url)}>Learn More</Button>
+            {item.regulationsurl && (
+              <Button size="small" onClick={() => openUrl(item.regulationsurl)}>
```

Comment:
Is it possible to add key to the button? It might be considered as a good practice since it's going to be part of rendering the button multiple times? Attaching the code snippet below, if that helps
`<Button key={item.id} size="small" onClick={() => openUrl(item.regulationsurl)}>
      Learn More
  </Button>`